code/modules/mob/living/living.dm

```diff
@@ -873,7 +873,7 @@ Thanks.
 	else if(istype(src.loc, /obj/item/delivery/large)) //Syndie item
 		var/obj/item/delivery/large/package = src.loc
 		to_chat(L, "<span class='warning'>You attempt to unwrap yourself, this package is tight and will take some time.</span>")
-		if(do_after(src, src, 100))
+		if(do_after(L, package, 2 MINUTES))
```

Comment:
NEVERMIND, this will make it so it's interrupted by moving the package, which is unwanted, I think.